app/hooks/useSearchForm.ts

```diff
@@ -1,16 +1,15 @@
+import { useSearchParams } from '@remix-run/react';
 import { ChangeEvent, FormEvent, useEffect, useState } from 'react';
-import { useLocation, useNavigate } from 'react-router-dom';
 import { useLocalStorage } from './useLocalStorage';
 
 export const useSearchForm = (inputRef: React.RefObject<HTMLInputElement>) => {
```

Comment:
я против таких записей: React.RefObject<HTMLInputElement
```suggestion
RefObject
```